### PR TITLE
Stef jun19

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Hi @jules32. As discussed, I've spent 2hrs on Lesson Series review:
- this PR
- opening issues
- tried to make pdf output (needs work) https://github.com/Openscapes/series/issues/44

File: how-do-i/discuss-community.qmd [hunk 1]
```diff
@@ -20,13 +21,15 @@ https://indigenous-data-envs-2022.netlify.app/index.html
...
 
 How do you talk to others (managers, supervisors) about the importance of community building and building social infrastructure? Here are some starting resources for reading about community building with Openscapes.
 
 They are in order of article length and increasing specificity:
 
+-   [Shifting institutional culture to develop climate solutions with Open Science](https://onlinelibrary.wiley.com/doi/10.1002/ece3.11341) (Lowndes et al 2024) - An Open Access Editorial written collaboratively by Mentors across the Openscapes community
+
 -   [Supercharge your research: a ten-week plan for open data science](https://openscapes.github.io/supercharge-research/) (Lowndes et al 2019) - community building within teams, based on our flagship program Openscapes Champions
 
 -   [Shifting institutional culture to develop climate solutions with Open Science](https://eartharxiv.org/repository/view/5948/) (Lowndes et al 2023) - community building within & across government orgs; authors are Openscapes Mentors who collaborate & support their colleagues within orgs
```
Comment (jules32) [inline how-do-i/discuss-community.qmd:35]:
```suggestion
```

File: inspiration/resources-that-influence.qmd [hunk 2]
```diff
@@ -88,48 +86,44 @@ https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008770
...
-8. The strength of negative feedback loops, relative to the impacts they are trying to correct against.
-7. The gain around driving positive feedback loops.
-6. The structure of information flows (who does and does not have access to information).
-5. The rules of the system (such as incentives, punishments, constraints).
-4. The power to add, change, evolve, or self-organize system structure.
-3. The goals of the system.
-2. The mindset or paradigm out of which the system — its goals, structure, rules, delays, parameters — arises.
-1. The power to transcend paradigms.
-
+13. The sizes of buffers and other stabilizing stocks, relative to their flows.
```
Comment (jules32) [inline inspiration/resources-that-influence.qmd:94]: I think this edit is not correct; the numbered list goes from 12 down to 1 not 12 to 23

**Review by jules32 — APPROVED** [review]
Thanks for these edits Stef, merging! I will fix the Donella Meadows numbering

**Pull request merged**